axelor-human-resource/src/main/resources/domains/EmployeeFile.xml

```diff
@@ -12,6 +12,18 @@
     <date name="recordDate" title="Record date"/>
     <many-to-one name="employee" ref="com.axelor.apps.hr.db.Employee"/>
     <date name="expirationDate" title="Expiration date"/>
+    <string name="fullName" namecolumn="true" search="fileType,metaFile" title="Full name">
+    	<![CDATA[
+    		String fullName = "";
+    		if(fileType != null){
+    			fullName = fileType.getName();
+    		}
+    		if(metaFile != null){
```

Comment:
please also check here if `fileType.getName()` and `metaFile.getFileName()` are not null, we need to be safe with getter methods since the method is often called.